spec/dummy/config/application.rb

```diff
@@ -61,6 +61,7 @@ class Application < Rails::Application
     # HACK: allows the fonts to work despite the namespaced bower_components
     config.assets.paths << Rails.root.join('..', '..', 'vendor', 'assets', 'bower_components')
 
+    config.time_zone = 'London'
```

Comment:
I added this config to the dummy app as `Time.now` was on GMT, but `Time.current` was on UTC time. this made debugging tests slightly more confusing.